pydm/widgets/archiver_time_plot.py

```diff
@@ -604,15 +610,6 @@ def redrawCurve(self, min_x=None, max_x=None) -> None:
         """
         Redraw the curve with any new data added since the last draw call.
         """
-        if not self.pvs:
-            # If we are just a constant, then forget about data
```

Comment:
can we keep these comments? provides context that we are handling constant case and about the bug

Reply:
Added these comments back in and moved them to FormulaCurveItem.evaluate with the other changes.